x/dex/keeper/core.go

```diff
@@ -497,7 +473,7 @@ func (k Keeper) CancelLimitOrderCore(goCtx context.Context, msg *types.MsgCancel
 			return err
 		}
 	} else {
-		return sdkerrors.Wrapf(types.ErrCannotWithdrawLimitOrder, "Cannot cancel liqudity from this limit order at this time")
+		return sdkerrors.Wrapf(types.ErrCancelEmptyLimitOrder, "Cannot cancel non-positive limit order amount")
```

Comment:
Do you know why we need these Wrapf messages if the message and the error's message are the same?